sphinx_scality/static/css/theme.css

```diff
@@ -3,3 +3,122 @@
 
 @import url("./fonts.css");
 @import url("./current.css");
+
+
+/****** header******/
```

Comment:
Comments syntax is `/* <message> */`, let's try to be consistent and not add a random number of stars. Welcome to the rigid world of development ;)